worldedit-core/src/main/java/com/fastasyncworldedit/core/function/generator/CavesGen.java

```diff
@@ -189,11 +189,11 @@ protected void generateCaveNode(
                 n = 16;
             }
 
-            if (i1 < 1) {
-                i1 = 1;
+            if (i1 < chunk.getMinY() + 1) {
+                i1 = chunk.getMinY();
             }
-            if (i2 > 256 - 8) {
-                i2 = 256 - 8;
+            if (i2 > chunk.getMaxY() - 8) {
```

Comment:
If `getMaxY` is inclusive, the old code is either wrong or a  `+ 1` is needed

Reply:
Yeah. All the inclusive and exclusive stuff needs to be clarified all over it looks like.

Reply:
This is a resolved, outdated comment 🙄

Reply:
I only just finished my review a wrote a couple of days ago. Sry